### PR TITLE
Sync common protos packages

### PR DESCRIPTION
**Pull request by shinfan**

### CODE REVIEW

**Comment by shinfan** [comment]
@garrettjonesgoogle  I have removed servicemanager/control PTAL

File: generated/java/proto-google-common-protos/src/main/proto/google/api/servicemanagement/v1/servicemanager.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Google Inc.
+// Copyright 2017 Google Inc.
```
Comment (garrettjonesgoogle) [inline generated/java/proto-google-common-protos/src/main/proto/google/api/servicemanagement/v1/servicemanager.proto]: Why is the proto still included?

Comment (shinfan) [inline generated/java/proto-google-common-protos/src/main/proto/google/api/servicemanagement/v1/servicemanager.proto]: Done

**Comment by shinfan** [comment]
@garrettjonesgoogle PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**